sources/packages/api/src/esdc-integration/msfaa-integration/msfaa-integration.service.ts

```diff
@@ -149,7 +149,7 @@ export class MSFAAIntegrationService {
       filesToProcess = await client.list(
         `${this.esdcConfig.ftpResponseFolder}`,
         new RegExp(
-          `^${this.esdcConfig.environmentCode}EDU.PBC.MSFA.REC.*\\.DAT`,
+          `^${this.esdcConfig.environmentCode}EDU.PBC.MSFA.REC.*`,
```

Comment:
this is the response file logic, right? do we remove it from here?

Reply:
Yes this is on the response file consuming